slick/src/main/scala/slick/jdbc/OracleProfile.scala

```diff
@@ -171,7 +171,7 @@ trait OracleProfile extends JdbcProfile {
 BEGIN
 
 """+ ((createPhase1 ++ createPhase2).map{s =>
-      "execute immediate '"+ s.replaceAll("'", """\\'""") + " ';"
+      "execute immediate '"+ s.replaceAll("'", """''""") + " ';"
```

Comment:
Since the escape no longer contains backslashes (\ ), there is no need to use the triple quotes ("""). Please change.

I don't know anything about Oracle, so can't verify if `'` should be quoted as `''`. Is there some documentation (or help forum article) you can point me to verify that.

Reply:
Changed.

There are two possible ways to do it, both covered [here](https://stackoverflow.com/a/11315369/495796).